### ISSUE TITLE
Lack of Visible Focus - WCAG Success Criterion 2.4.7

### ISSUE THREAD

**Issue by twinfred** [HospitalRun/hospitalrun-frontend#2653]
## 🐛 Bug Report

When tabbing through the site, there is no visible focus for keyboard users. This violates [WCAG Success Criterion 2.4.7](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-focus-visible.html).

## To Reproduce

Use a keyboard to navigate through the site. You will notice that there is no visible focus indicator as you navigate, providing no context regarding focus location to keyboard users. The native browser focus ring is being suppressed in the CSS using `outline: 0`.

## Expected behavior

When tabbing through the site, the location of the keyboard focus should be obvious. Removing the `outline: 0` from the CSS should allow the browser's native focus ring to appear.

## Your Environment

- *node version*: 12.19.0
- *fastify version*: >=1.0.0
- *os*: Windows
- *any other relevant information*: Chrome desktop web browser

**Comment by twinfred** [comment]
I will be submitting a fix for this bug today :)
